packages/client/src/utils/transition.js

```diff
@@ -0,0 +1,16 @@
+import { fade, blur, scale, fly } from "svelte/transition"
+
+// Default options
+const transitions = new Map([
+  ["fade", { tn: fade, opt: {} }],
+  ["blur", { tn: blur, opt: {} }],
+  // This one seems to not result in any effect
+  // ["slide", { tn: slide, opt: {} }],
+  ["scale", { tn: scale, opt: {} }],
+  ["fly", { tn: fly, opt: { y: 80 } }],
+])
+
+export default function transition(node, { type, options = {} }) {
+  const { tn, opt } = transitions.get(type) || {}
+  return tn ? tn(node, { ...opt, ...options }) : fade(node, { duration: 0 })
```

Comment:
I'm assuming the fallback here of `fade` with `duration: 0` just does nothing - i.e. no transition? I'm sure performance difference is minimal but just considering there could be thousands of containers in a page, it might be better to explicitly return null, or a NOP, or whatever is valid as a svelte transition with the smallest runtime logic.

Reply:
Yep. That's the reason. I pondered this a bit and came to the conclusion that it shouldn't matter much but I can just as well replace it with an inline function.